other/check_strong_password.py

```diff
@@ -0,0 +1,47 @@
+# This Will Check Whether A Given Password Is Strong Or Not
+# It Follows The Rule that Length Of Password Should Be At Least 8 Characters
+# And At Least 1 Lower, 1 Upper, 1 Number And 1 Special Character
+
+
+import re
+
+
+def strong_password_detector(password: str) -> str:
```

Comment:
```suggestion
def strong_password_detector(password: str, min_length: int = 8) -> str:
```
and make appropriate changes in the function.